Mathlib/Analysis/Normed/Operator/LinearIsometry.lean

```diff
@@ -170,14 +170,6 @@ theorem ext {f g : E →ₛₗᵢ[σ₁₂] E₂} (h : ∀ x, f x = g x) : f = g
 
 variable [FunLike 𝓕 E E₂]
 
-protected theorem congr_arg {f : 𝓕} :
-    ∀ {x x' : E}, x = x' → f x = f x'
-  | _, _, rfl => rfl
-
-protected theorem congr_fun {f g : 𝓕} (h : f = g) (x : E) :
-    f x = g x :=
-  h ▸ rfl
```

Comment:
Why did you remove these ones?

Reply:
Hmm, they were unused, inconvenient to move to a place where the variables would be suitable, and hopefully are taken care of by `FunLike.congr_arg`. I could go through the deprecation hoops, but I'm willing to cut a few corners getting this project out of the way. :-)

Reply:
ok, fine for these!